bundle-workflow/src/run_integ_test.py

```diff
@@ -40,6 +41,7 @@ def main():
             integ_test_config[component.name] = component
     with TemporaryDirectory(keep=args.keep) as work_dir:
         logging.info("Switching to temporary work_dir: " + work_dir)
+        TestRecorder(args.test_run_id, "integ-test", work_dir)
```

Comment:
This looks odd, we create and throw away an instance of `TestRecorder`. I'd expect to call some method here.

Reply:
Its just instantiating the test_recorder at the test_suite level. No methods are used but the same instance is retrieved for calling methods in `local_test_cluster` and `integ_test_suite`